templates/vue-starter-template/app/components/FrontendDetailPage.vue

```diff
@@ -0,0 +1,87 @@
+<script setup lang="ts">
+import { getProductName } from "@shopware/helpers";
+
+const props = defineProps<{
+  navigationId: string;
+}>();
+
+const { search } = useProductSearch();
+const { buildDynamicBreadcrumbs, pushBreadcrumb } = useBreadcrumbs();
+const { apiClient } = useShopwareContext();
+const errorDetails = ref();
+
+const { data, error } = await useAsyncData(
+  `cmsProduct${props.navigationId}`,
+  async () => {
+    const responses = await Promise.allSettled([
+      search(props.navigationId, {
+        withCmsAssociations: true,
+        associations: {
+          seoUrls: {},
+        },
+      }),
+      apiClient.invoke("readBreadcrumb get /breadcrumb/{id}", {
+        pathParams: {
+          id: props.navigationId,
+        },
+      }),
+    ]);
+
+    for (const response of responses) {
+      if (response.status === "rejected") {
+        console.error("[FrontendDetailPage.vue]", response.reason.message);
+        errorDetails.value = response.reason.message;
+      }
+    }
+
+    return {
+      productResponse:
+        responses[0].status === "fulfilled" ? responses[0].value : null,
+      breadcrumbs:
+        responses[1].status === "fulfilled" ? responses[1].value : null,
+    };
+  },
+);
+const productResponse = ref(data.value?.productResponse);
```

Comment:
`data.value` is never null here, we do not need optional chaining for it


additionally we do not replace this value, I would avoid rewriting that to separate ref, just use it everywhere by the reference, or even just put it into the const

```suggestion
const productResponse = data.value.productResponse;
```

Reply:
done